If installing to "/usr", do not embed RPATH entries in SDL shared libraries;
otherwise, do. (RPATH is demonstrably terrible, unless absolutely needed on
Gentoo Prefix and/or for cross-compilation.)

--- sdl2-config.in	2012-08-22 23:22:01.344325854 -0500
+++ sdl2-config.in.new	2012-08-25 17:34:36.573610492 -0500
@@ -45,7 +45,9 @@
       echo -I@includedir@/SDL2 @SDL_CFLAGS@
       ;;
 @ENABLE_SHARED_TRUE@    --libs)
-@ENABLE_SHARED_TRUE@      echo -L@libdir@ @SDL_RLD_FLAGS@ @SDL_LIBS@
+@ENABLE_SHARED_TRUE@      libdirs=
+@ENABLE_SHARED_TRUE@      test x"${prefix}" == x"/usr" || libdirs="-L@libdir@"
+@ENABLE_SHARED_TRUE@      echo ${libdirs} @SDL_RLD_FLAGS@ @SDL_LIBS@
 @ENABLE_SHARED_TRUE@      ;;
 @ENABLE_STATIC_TRUE@@ENABLE_SHARED_TRUE@    --static-libs)
 @ENABLE_STATIC_TRUE@@ENABLE_SHARED_FALSE@    --libs|--static-libs)
